.github/workflows/ci.yaml

```diff
@@ -41,8 +41,8 @@ jobs:
         os:
           # temporarily disable non-windows builds to speed up iteration time
           - ubuntu-20.04
-          - macOS-11.0
-          - windows-2019
+          # - macOS-11.0
+          # - windows-2019
```

Comment:
We aren't going to run CI on macOS/windows any more?

Also if you change this can you remove the misleading comment?

Reply:
This is just temporary while I work it out! But yeah I can change the comment